src/ros_vrpn_client.cpp

```diff
@@ -29,103 +29,125 @@
  ## author Chih-Hung Aaron King (Healthcare Robotics Lab, Georgia Tech.)
  */
 
-//== This application listens for a rigid body named 'Tracker' on a remote machine
-//== and publishes & tf it's position and orientation through ROS.
-#include <ros/ros.h>
-#include <tf/transform_broadcaster.h>
-#include <geometry_msgs/TransformStamped.h>
+// This application listens for a rigid body named 'Tracker' on a remote machine.
+// The raw data is input to a Extended Kalman Filter (EKF) based estimator estimating
+// the target position, velocity, orientation and angular velocity. The estimated
+// quantities and raw data are then published through ROS.
+
 #include <iostream>
-#include <nav_msgs/Odometry.h>
 #include <stdio.h>
 #include <math.h>
 #include <iostream>
 
+#include <ros/ros.h>
 #include <vrpn_Connection.h>
 #include <vrpn_Tracker.h>
 #include <Eigen/Geometry>
 #include <eigen_conversions/eigen_msg.h>
+#include <nav_msgs/Odometry.h>
+#include <geometry_msgs/TransformStamped.h>
+#include <tf/transform_broadcaster.h>
+#include <glog/logging.h>
 
 #include "vicon_odometry_estimator.h"
 
-void VRPN_CALLBACK track_target(void *, const vrpn_TRACKERCB t);
+void VRPN_CALLBACK track_target(void *, const vrpn_TRACKERCB tracker);
 
-// TODO(millanea@ethz.ch): The following callbacks should be implemented in the case that
+// NOTE(millanea@ethz.ch): The following callbacks should be implemented in the case that
 //                         the tracking system supports them. In this case a flag should be
 //                         added to the code indicating the whether or not the vicon estimator
 //                         should be run.
 //void VRPN_CALLBACK track_target_velocity(void *, const vrpn_TRACKERVELCB tv);
 //void VRPN_CALLBACK track_target_acceleration(void *, const vrpn_TRACKERACCCB ta);
 
+// A class representing the state of the tracked target.
 class TargetState
```

Comment:
Probably more stylistically correct to have this be a struct, since it just has public members and nothing else.